cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis_without_creds.test.ts

```diff
@@ -84,6 +88,34 @@ describe(["@tier1"], "Source Analysis without credentials", () => {
         );
     });
 
+    // Automates Bug https://issues.redhat.com/browse/MTA-3440
+    it("Source analysis on bookserver app with EAP8 target", function () {
+        const Application = new Analysis(
+            getRandomApplicationData("eap8-bookserverApp", {
+                sourceData: this.appData["bookserver-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["EAP8_bookserverapp"])
+        );
+        Analysis.open();
+        Application.create();
+        applicationsList.push(Application);
+        cy.wait("@getApplication");
+        cy.wait(2000);
+        Application.analyze();
+        checkSuccessAlert(infoAlertMessage, `Submitted for analysis`);
+        cy.get(tdTag, { log: false })
+            .contains(Application.name)
+            .closest(trTag, { log: false })
+            .within(() => {
+                Analysis.verifyStatus(
+                    cy
+                        .get(analysisColumn, { log: false })
+                        .should("not.have.text", "Completed with Errors"),
+                    "Completed"
```

Comment:
You can replace the text with `AnalysisStatuses.completed` to aim for consistency